AV/VehicleAutomationLevels.rdf

```diff
@@ -582,11 +668,75 @@
+		<rdf:type rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/WeatherType"/>
+		<rdfs:label xml:lang="en">foggy weather</rdfs:label>
+		<skos:definition xml:lang="en">Weather characterized by fog that reduces visibility on the road.</skos:definition>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/Freeway">
+		<rdf:type rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/RoadType"/>
+		<rdfs:label xml:lang="en">freeway</rdfs:label>
+		<skos:definition xml:lang="en">A wide road for fast-moving traffic, especially in the US, with a limited number of places at which drivers can enter and leave it.</skos:definition>
+	</owl:NamedIndividual>
+	
+	<owl:Class rdf:about="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/FreewaysLowSpeedOptimalRoadMaintenanceFairWeather">
+		<rdfs:subClassOf rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/OperationalDesignDomain"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/hasOperatingCondition"/>
+				<owl:hasValue rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/FairWeather"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/hasOperatingCondition"/>
+				<owl:hasValue rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/Freeway"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/hasOperatingCondition"/>
+				<owl:hasValue rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/OptimalRoadMaintenance"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/hasOperatingCondition"/>
+				<owl:someValuesFrom rdf:resource="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/LowSpeedLimit"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">freeways, low speed, optimal road maintenance, fair weather</rdfs:label>
+		<skos:definition xml:lang="en">Operating conditions: fully access-controlled freeways in low-speed traffic under fair weather conditions and optimal road maintenance.</skos:definition>
+		<AnnotationVocabulary:adaptedFrom xml:lang="en">SAE-J3016 (2018), sec. 3.22, Example 1</AnnotationVocabulary:adaptedFrom>
+	</owl:Class>
+	
+	<owl:NamedIndividual rdf:about="https://spec.edmcouncil.org/auto/ontology/AV/VehicleAutomationLevels/FreewaysTrafficJams">
```

Comment:
An instance of CorporateCampusSpecificRoute class is an ODD covering a specific road in a specific campus. Such instances can be used in descriptions of specific vehicle features. On the other hand, the ODD FreewaysTrafficJams includes all freeways (and traffic jams on them), not just a selected freeway. Associating such ODDs (defined as classes) with specific vehicle features (defined as instances) requires representing classes as object property values. I wanted to avoid this.